src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -26,13 +26,13 @@ public class SortCommandParser implements Parser<SortCommand> {
     public static final String SORT_COMPANIES_ARG_WORD = "c";
 
     /**
-     * Regex used to confirm if the arguments are either c or p for sort command.
+     * Regex used to confirm if the arguments is p for sort command.
```

Comment:
I think you may have accidentally used `p` here instead of `c`. (Same on line 35.)